### PR TITLE
UAG-158 - Fix: Assets not regenerated for frontend when post conatining reusable blocks is updated.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
Issue - https://ua-gutenberg.atlassian.net/jira/software/projects/UAG/boards/2?selectedIssue=UAG-158

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
 - Added test cases in the issue.

### Checklist:
- [x] My code is tested
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-admin.php [hunk 1]
```diff
@@ -700,6 +700,21 @@ public static function admin_submenu_css() {
 		 */
 		public static function delete_page_assets( $post_id ) {
 
+			$does_post_contain_reusable_blocks = self::does_post_contain_reusable_blocks( $post_id );
+
+			if ( true === $does_post_contain_reusable_blocks ) {
+
+				if ( 'enabled' === UAGB_Helper::$file_generation ) {
```
Comment (sandeshjangam) [inline classes/class-uagb-admin.php:707]: Remove 707 to 710

File: classes/class-uagb-admin.php [hunk 2]
```diff
@@ -700,6 +700,21 @@ public static function admin_submenu_css() {
...
+
+					UAGB_Helper::delete_all_uag_dir_files();
+				}
+
+				/* Update the asset version */
+				update_option( '__uagb_asset_version', time() );
+
+				return;
+			}
```
Comment (sandeshjangam) [inline classes/class-uagb-admin.php:716]: Add this whole condition after delete_post_meta. Line 737

File: classes/class-uagb-admin.php (hunk 1 above)
Comment (RajkiranBagal) [reply classes/class-uagb-admin.php]: Done

File: classes/class-uagb-admin.php (hunk 2 above)
Comment (RajkiranBagal) [reply classes/class-uagb-admin.php]: Done

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**